cypress/e2e/tests/migration/task-manager/sorting.filter.pagination.test.ts

```diff
@@ -0,0 +1,177 @@
+        validateTextPresence(taskManagerColumns.application, applicationsList[0].name);
+        validateTextPresence(taskManagerColumns.application, applicationsList[1].name, false);
+        validateTextPresence(taskManagerColumns.kind, "analyzer");
+        validateTextPresence(taskManagerColumns.kind, "language-discovery");
+        validateTextPresence(taskManagerColumns.kind, "tech-discovery");
+        validateTextPresence(taskManagerColumns.status, "Running");
+        validateTextPresence(taskManagerColumns.status, "Succeeded");
+        clearAllFilters();
+
+        TaskManager.applyAndValidateFilter(taskFilter.applicationName, applicationsList[1].name);
+        validateTextPresence(taskManagerColumns.application, applicationsList[1].name);
+        validateTextPresence(taskManagerColumns.kind, "analyzer");
+        validateTextPresence(taskManagerColumns.kind, "language-discovery");
+        validateTextPresence(taskManagerColumns.kind, "tech-discovery");
+        clearAllFilters();
+
+        // Filter by Kind
+        TaskManager.applyAndValidateFilter(taskFilter.kind, "analyzer");
+        validateTextPresence(taskManagerColumns.kind, "analyzer");
+        validateTextPresence(taskManagerColumns.kind, "language-discovery", false);
+        validateTextPresence(taskManagerColumns.kind, "tech-discovery", false);
+        clearAllFilters();
+        TaskManager.applyAndValidateFilter(taskFilter.kind, "language-discovery");
+        validateTextPresence(taskManagerColumns.kind, "language-discovery");
+        validateTextPresence(taskManagerColumns.kind, "analyzer", false);
+        validateTextPresence(taskManagerColumns.kind, "tech-discovery", false);
+        clearAllFilters();
+        TaskManager.applyAndValidateFilter(taskFilter.kind, "tech-discovery");
+        validateTextPresence(taskManagerColumns.kind, "tech-discovery");
+        validateTextPresence(taskManagerColumns.kind, "analyzer", false);
+        validateTextPresence(taskManagerColumns.kind, "language-discovery", false);
+        clearAllFilters();
+
+        // Filter by Created By
+        TaskManager.applyAndValidateFilter(taskFilter.createdBy, "admin");
+        validateTextPresence(taskManagerColumns.createdBy, "admin");
+        clearAllFilters();
+
+        // Negative test, filtering by not existing data
+        TaskManager.applyAndValidateFilter(taskFilter.applicationName, randomWordGenerator(6));
+        cy.get(trTag).should("contain", "No results found");
+        clearAllFilters();
+    });
+
+    it("Sorting tasks", function () {
+        TaskManager.open();
+        sortByList.forEach((column) => {
+            validateSortBy(column);
+        });
+    });
+
+    it("Pagination validation", function () {
+        TaskManager.open(10);
+        validatePagination();
+    });
+
+    after("Perform test data clean up", function () {
+        cy.reload();
+        cy.log("Deleting app list");
```

Comment:
The `cy.reload()` call is a really good idea!

The `cy.log()` line can be removed.